charts/provisioner-config-local/config/pp-deploy-cp-core-on-prem.yaml

```diff
@@ -26,7 +26,7 @@ options:
   guiType: input
   reference: "meta.guiEnv.GUI_CP_CHART_REPO"
   description: |
-    The helm chart repository to deploy CP. The default value is public repo. To use private repo set: https://${GITHUB_TOKEN}@raw.githubusercontent.com/tibco/tp-helm-charts/gh-pages
+    The helm chart repository to deploy CP. The default value is public repo. To use private repo set: https://${GUI_CP_CHART_REPO}@raw.githubusercontent.com/tibco/tp-helm-charts/gh-pages
```

Comment:
@syan-tibco : is this expected ? We don't need GITHUB_TOKEN anymore for private repo ?